### PR TITLE
Add the ability to specify an http(s) URL for config

### PR DESCRIPTION
**Pull request by sudermanjr**

### CODE REVIEW

**Review by makoscafee — APPROVED** [review]

File: cmd/root.go [hunk 1]
```diff
@@ -84,6 +90,16 @@ func initConfig() {
 		return
 	}
 
+	if strings.Contains(cfgFile, "http") {
```
Comment (rbren) [inline cmd/root.go:93]: I'd suggest being a little more strict here. Here's polaris' code: https://github.com/FairwindsOps/polaris/blob/ef4f5aeaebaaa804caf3e241774edb7df8471ee0/pkg/config/config.go#L62

Comment (sudermanjr) [reply cmd/root.go:93]:
```suggestion
	if strings.HasPrefix(cfgFile, "https://") || strings.HasPrefix(cfgFile, "http://") {
```

File: cmd/root.go [hunk 2]
```diff
@@ -92,6 +108,43 @@ func initConfig() {
...
+func downloadConfig(cfgURL string) (string, error) {
+	fileURL, err := url.Parse(cfgURL)
+	if err != nil {
+		return "", err
+	}
+	path := fileURL.Path
+	segments := strings.Split(path, "/")
+	fileName := segments[len(segments)-1]
+
+	file, err := ioutil.TempFile("", fmt.Sprintf("*-%s", fileName))
```
Comment (rbren) [inline cmd/root.go:120]: Neat, I didn't know this was a thing

**Review by rbren — APPROVED** [review]

File: cmd/root.go (hunk 2 above)
Comment (sudermanjr) [reply cmd/root.go:120]: Just discovered it recently. Used it in the new Reckoner code. It's pretty convenient.

**Pull request merged**